src/common/config.go

```diff
@@ -248,6 +255,16 @@ func checkConf() error {
 		return fmt.Errorf("Unknown Sandbox type '%s'", Conf.Sandbox)
 	}
 
+	if Conf.Trace.Format != "default" {
+		if Conf.Trace.Format == "text" || Conf.Trace.Format == "json" {
+			if Conf.Trace.Log_file_dir == "" {
```

Comment:
If this is specified, can't we just send the log output to stdout?

Reply:
default to the stdout, else json to a file